gossip/Cargo.toml

```diff
@@ -17,6 +17,7 @@ crossbeam-channel = "0.5"
 flate2 = "1.0"
 indexmap = { version = "1.8", features = ["rayon"] }
 itertools = "0.10.3"
+regex = "1"
```

Comment:
Can you please move this to dev-dependencies, since it's only used in a test?

Reply:
Sure.